[2020-02-02T00:09:00.327] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"pages":29,"message":"hello world !!!"}} - ::1 - 4ms
[2020-02-02T00:22:19.921] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"pages":29,"message":"hello world !!!"}} - ::1 - 3ms
[2020-02-02T00:36:30.557] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"pages":29,"message":"hello world !!!"}} - ::1 - 3ms
[2020-02-02T01:12:01.776] [ERROR] [Default] - Error: MongoNetworkError: failed to connect to server [127.0.0.1:27017] on first connect [MongoError: Authentication failed.
    at Function._getError (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:141:14)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:191:31
    at _callback (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:320:5)
    at Connection.messageHandler (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:349:5)
    at Connection.emit (events.js:200:13)
    at processMessage (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:200:13)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at Socket.Readable.push (_stream_readable.js:210:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:166:17) {
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {}
}]
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:30:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:12:05.806] [ERROR] [Default] - Error: MongoNetworkError: failed to connect to server [127.0.0.1:27017] on first connect [MongoError: Authentication failed.
    at Function._getError (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:141:14)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:191:31
    at _callback (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:320:5)
    at Connection.messageHandler (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:349:5)
    at Connection.emit (events.js:200:13)
    at processMessage (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:200:13)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at Socket.Readable.push (_stream_readable.js:210:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:166:17) {
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {}
}]
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:30:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:12:06.393] [ERROR] [Default] - Error: MongoNetworkError: failed to connect to server [127.0.0.1:27017] on first connect [MongoError: Authentication failed.
    at Function._getError (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:141:14)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:191:31
    at _callback (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:320:5)
    at Connection.messageHandler (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:349:5)
    at Connection.emit (events.js:200:13)
    at processMessage (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:200:13)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at Socket.Readable.push (_stream_readable.js:210:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:166:17) {
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {}
}]
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:30:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:14:26.759] [ERROR] [Default] - Error: MongoNetworkError: failed to connect to server [127.0.0.1:27017] on first connect [MongoError: Authentication failed.
    at Function._getError (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:141:14)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:191:31
    at _callback (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:320:5)
    at Connection.messageHandler (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:349:5)
    at Connection.emit (events.js:200:13)
    at processMessage (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:200:13)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at Socket.Readable.push (_stream_readable.js:210:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:166:17) {
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {}
}]
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:30:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:17:58.687] [ERROR] [Default] - Error: MongoNetworkError: failed to connect to server [127.0.0.1:27017] on first connect [MongoError: Authentication failed.
    at Function._getError (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:141:14)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:191:31
    at _callback (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:320:5)
    at Connection.messageHandler (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:349:5)
    at Connection.emit (events.js:200:13)
    at processMessage (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:200:13)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at Socket.Readable.push (_stream_readable.js:210:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:166:17) {
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {}
}]
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:30:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:18:00.661] [ERROR] [Default] - Error: MongoNetworkError: failed to connect to server [127.0.0.1:27017] on first connect [MongoError: Authentication failed.
    at Function._getError (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:141:14)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:191:31
    at _callback (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:320:5)
    at Connection.messageHandler (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:349:5)
    at Connection.emit (events.js:200:13)
    at processMessage (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:200:13)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at Socket.Readable.push (_stream_readable.js:210:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:166:17) {
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {}
}]
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:30:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:21:56.939] [ERROR] [Default] - Error: MongoNetworkError: failed to connect to server [127.0.0.1:27017] on first connect [MongoError: Authentication failed.
    at Function._getError (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:141:14)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:191:31
    at _callback (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:320:5)
    at Connection.messageHandler (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:349:5)
    at Connection.emit (events.js:200:13)
    at processMessage (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:200:13)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at Socket.Readable.push (_stream_readable.js:210:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:166:17) {
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {}
}]
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:30:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:22:01.052] [ERROR] [Default] - Error: MongoNetworkError: failed to connect to server [127.0.0.1:27017] on first connect [MongoError: Authentication failed.
    at Function._getError (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:141:14)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:191:31
    at _callback (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:320:5)
    at Connection.messageHandler (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:349:5)
    at Connection.emit (events.js:200:13)
    at processMessage (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:200:13)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at Socket.Readable.push (_stream_readable.js:210:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:166:17) {
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {}
}]
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:30:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:22:01.745] [ERROR] [Default] - Error: MongoNetworkError: failed to connect to server [127.0.0.1:27017] on first connect [MongoError: Authentication failed.
    at Function._getError (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:141:14)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:191:31
    at _callback (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:320:5)
    at Connection.messageHandler (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:349:5)
    at Connection.emit (events.js:200:13)
    at processMessage (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:200:13)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at Socket.Readable.push (_stream_readable.js:210:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:166:17) {
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {}
}]
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:30:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:22:17.180] [ERROR] [Default] - Error: MongoNetworkError: failed to connect to server [127.0.0.1:27017] on first connect [MongoError: Authentication failed.
    at Function._getError (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:141:14)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:191:31
    at _callback (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:320:5)
    at Connection.messageHandler (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:349:5)
    at Connection.emit (events.js:200:13)
    at processMessage (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:200:13)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at Socket.Readable.push (_stream_readable.js:210:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:166:17) {
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {}
}]
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:30:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:22:18.780] [ERROR] [Default] - Error: MongoNetworkError: failed to connect to server [127.0.0.1:27017] on first connect [MongoError: Authentication failed.
    at Function._getError (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:141:14)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:191:31
    at _callback (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:320:5)
    at Connection.messageHandler (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:349:5)
    at Connection.emit (events.js:200:13)
    at processMessage (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:200:13)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at Socket.Readable.push (_stream_readable.js:210:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:166:17) {
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {}
}]
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:30:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:22:19.519] [ERROR] [Default] - Error: MongoNetworkError: failed to connect to server [127.0.0.1:27017] on first connect [MongoError: Authentication failed.
    at Function._getError (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:141:14)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:191:31
    at _callback (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:320:5)
    at Connection.messageHandler (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:349:5)
    at Connection.emit (events.js:200:13)
    at processMessage (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:200:13)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at Socket.Readable.push (_stream_readable.js:210:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:166:17) {
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {}
}]
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:30:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:25:02.210] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:25:35.715] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"pages":29,"message":"hello world !!!"}} - ::1 - 3ms
[2020-02-02T01:25:58.923] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:26:30.733] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
    at runNextTicks (internal/process/task_queues.js:59:3)
    at processTimers (internal/timers.js:472:9)
[2020-02-02T01:30:52.342] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:34:24.342] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:37:24.706] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:40:21.240] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:41:44.089] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:44:41.560] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:45:36.339] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:48:52.239] [ERROR] [Default] - TypeError: userService.getUserList is not a function
    at getUserList (/Users/edz/Desktop/resource/zhblog/controllers/category.js:15:44)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:353:16
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:358:31)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:339:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-static-cache/index.js:62:42
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/edz/Desktop/resource/zhblog/node_modules/koa-bodyparser/index.js:86:11)
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
    at async loggerMiddleware (/Users/edz/Desktop/resource/zhblog/middlewares/logger.js:48:5)
[2020-02-02T01:48:52.241] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":-1,"data":null,"msg":"userService.getUserList is not a function"} - ::1 - 4ms
[2020-02-02T01:49:03.267] [ERROR] [Default] - TypeError: userService.getUserList is not a function
    at getUserList (/Users/edz/Desktop/resource/zhblog/controllers/category.js:15:44)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:353:16
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:358:31)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:339:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-static-cache/index.js:62:42
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/edz/Desktop/resource/zhblog/node_modules/koa-bodyparser/index.js:86:11)
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
    at async loggerMiddleware (/Users/edz/Desktop/resource/zhblog/middlewares/logger.js:48:5)
[2020-02-02T01:49:03.267] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":-1,"data":null,"msg":"userService.getUserList is not a function"} - ::1 - 0ms
[2020-02-02T01:51:08.973] [ERROR] [Default] - TypeError: userService.getUserList is not a function
    at getUserList (/Users/edz/Desktop/resource/zhblog/controllers/category.js:15:44)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:353:16
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:358:31)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:339:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-static-cache/index.js:62:42
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/edz/Desktop/resource/zhblog/node_modules/koa-bodyparser/index.js:86:11)
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
    at async loggerMiddleware (/Users/edz/Desktop/resource/zhblog/middlewares/logger.js:48:5)
[2020-02-02T01:51:08.975] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":-1,"data":null,"msg":"userService.getUserList is not a function"} - ::1 - 4ms
[2020-02-02T01:51:10.334] [ERROR] [Default] - TypeError: userService.getUserList is not a function
    at getUserList (/Users/edz/Desktop/resource/zhblog/controllers/category.js:15:44)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:353:16
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:358:31)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:339:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-static-cache/index.js:62:42
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/edz/Desktop/resource/zhblog/node_modules/koa-bodyparser/index.js:86:11)
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
    at async loggerMiddleware (/Users/edz/Desktop/resource/zhblog/middlewares/logger.js:48:5)
[2020-02-02T01:51:10.334] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":-1,"data":null,"msg":"userService.getUserList is not a function"} - ::1 - 1ms
[2020-02-02T01:51:38.688] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:51:57.507] [ERROR] [Default] - TypeError: userService.getUserList is not a function
    at getUserList (/Users/edz/Desktop/resource/zhblog/controllers/category.js:16:44)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:353:16
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:358:31)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:339:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-static-cache/index.js:62:42
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/edz/Desktop/resource/zhblog/node_modules/koa-bodyparser/index.js:86:11)
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
    at async loggerMiddleware (/Users/edz/Desktop/resource/zhblog/middlewares/logger.js:48:5)
[2020-02-02T01:51:57.509] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":-1,"data":null,"msg":"userService.getUserList is not a function"} - ::1 - 4ms
[2020-02-02T01:52:22.268] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:54:05.539] [ERROR] [Default] - TypeError: userService.getUserList is not a function
    at getUserList (/Users/edz/Desktop/resource/zhblog/controllers/category.js:16:44)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:353:16
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:358:31)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:339:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-static-cache/index.js:62:42
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/edz/Desktop/resource/zhblog/node_modules/koa-bodyparser/index.js:86:11)
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
    at async loggerMiddleware (/Users/edz/Desktop/resource/zhblog/middlewares/logger.js:48:5)
[2020-02-02T01:54:05.539] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":-1,"data":null,"msg":"userService.getUserList is not a function"} - ::1 - 1ms
[2020-02-02T01:55:27.971] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:57:08.029] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T01:57:18.346] [ERROR] [Default] - TypeError: userService.getUserList is not a function
    at getUserList (/Users/edz/Desktop/resource/zhblog/controllers/category.js:16:44)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:353:16
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:358:31)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:339:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-static-cache/index.js:62:42
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/edz/Desktop/resource/zhblog/node_modules/koa-bodyparser/index.js:86:11)
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
    at async loggerMiddleware (/Users/edz/Desktop/resource/zhblog/middlewares/logger.js:48:5)
[2020-02-02T01:57:18.348] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":-1,"data":null,"msg":"userService.getUserList is not a function"} - ::1 - 4ms
[2020-02-02T01:58:06.629] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
    at runNextTicks (internal/process/task_queues.js:59:3)
    at processTimers (internal/timers.js:472:9)
[2020-02-02T02:00:31.470] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:00:56.078] [ERROR] [Default] - TypeError: services.category.getUserList is not a function
    at getUserList (/Users/edz/Desktop/resource/zhblog/controllers/category.js:16:50)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:353:16
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:358:31)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:339:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-static-cache/index.js:62:42
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/edz/Desktop/resource/zhblog/node_modules/koa-bodyparser/index.js:86:11)
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
    at async loggerMiddleware (/Users/edz/Desktop/resource/zhblog/middlewares/logger.js:48:5)
[2020-02-02T02:00:56.081] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":-1,"data":null,"msg":"services.category.getUserList is not a function"} - ::1 - 7ms
[2020-02-02T02:01:16.233] [ERROR] [Default] - TypeError: services.category.getUserList is not a function
    at getUserList (/Users/edz/Desktop/resource/zhblog/controllers/category.js:16:50)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:353:16
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:358:31)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:339:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-static-cache/index.js:62:42
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/edz/Desktop/resource/zhblog/node_modules/koa-bodyparser/index.js:86:11)
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
    at async loggerMiddleware (/Users/edz/Desktop/resource/zhblog/middlewares/logger.js:48:5)
[2020-02-02T02:01:16.235] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":-1,"data":null,"msg":"services.category.getUserList is not a function"} - ::1 - 4ms
[2020-02-02T02:01:26.963] [ERROR] [Default] - TypeError: services.category.getUserList is not a function
    at getUserList (/Users/edz/Desktop/resource/zhblog/controllers/category.js:16:50)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:353:16
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:358:31)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:339:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-static-cache/index.js:62:42
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/edz/Desktop/resource/zhblog/node_modules/koa-bodyparser/index.js:86:11)
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
    at async loggerMiddleware (/Users/edz/Desktop/resource/zhblog/middlewares/logger.js:48:5)
[2020-02-02T02:01:26.963] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":-1,"data":null,"msg":"services.category.getUserList is not a function"} - ::1 - 1ms
[2020-02-02T02:01:42.108] [ERROR] [Default] - TypeError: services.category.getUserList is not a function
    at getUserList (/Users/edz/Desktop/resource/zhblog/controllers/category.js:16:50)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:353:16
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:358:31)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:339:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-static-cache/index.js:62:42
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/edz/Desktop/resource/zhblog/node_modules/koa-bodyparser/index.js:86:11)
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
    at async loggerMiddleware (/Users/edz/Desktop/resource/zhblog/middlewares/logger.js:48:5)
[2020-02-02T02:01:42.108] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":-1,"data":null,"msg":"services.category.getUserList is not a function"} - ::1 - 1ms
[2020-02-02T02:01:42.163] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:03:05.156] [ERROR] [Default] - TypeError: categoryService.getUserList is not a function
    at getUserList (/Users/edz/Desktop/resource/zhblog/controllers/category.js:16:48)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:353:16
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:358:31)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:339:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-static-cache/index.js:62:42
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/edz/Desktop/resource/zhblog/node_modules/koa-bodyparser/index.js:86:11)
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
    at async loggerMiddleware (/Users/edz/Desktop/resource/zhblog/middlewares/logger.js:48:5)
[2020-02-02T02:03:05.159] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":-1,"data":null,"msg":"categoryService.getUserList is not a function"} - ::1 - 4ms
[2020-02-02T02:04:15.576] [ERROR] [Default] - TypeError: categoryService.getCategoryList is not a function
    at getUserList (/Users/edz/Desktop/resource/zhblog/controllers/category.js:16:48)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:353:16
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:358:31)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:339:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-static-cache/index.js:62:42
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/edz/Desktop/resource/zhblog/node_modules/koa-bodyparser/index.js:86:11)
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
    at async loggerMiddleware (/Users/edz/Desktop/resource/zhblog/middlewares/logger.js:48:5)
[2020-02-02T02:04:15.579] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":-1,"data":null,"msg":"categoryService.getCategoryList is not a function"} - ::1 - 5ms
[2020-02-02T02:04:33.088] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:06:15.887] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
    at runNextTicks (internal/process/task_queues.js:59:3)
    at processTimers (internal/timers.js:472:9)
[2020-02-02T02:08:51.521] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
    at runNextTicks (internal/process/task_queues.js:59:3)
    at processTimers (internal/timers.js:472:9)
[2020-02-02T02:11:27.202] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:14:16.427] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
    at runNextTicks (internal/process/task_queues.js:59:3)
    at processTimers (internal/timers.js:472:9)
[2020-02-02T02:14:48.878] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:17:27.479] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:18:38.050] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:19:21.069] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:21:19.171] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:22:16.103] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:24:19.528] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:25:13.493] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
    at runNextTicks (internal/process/task_queues.js:59:3)
    at processTimers (internal/timers.js:472:9)
[2020-02-02T02:25:44.714] [ERROR] [Default] - Error: MongoNetworkError: failed to connect to server [127.0.0.1:27017] on first connect [MongoError: Authentication failed.
    at Function._getError (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:141:14)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:191:31
    at _callback (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:320:5)
    at Connection.messageHandler (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:349:5)
    at Connection.emit (events.js:200:13)
    at processMessage (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:200:13)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at Socket.Readable.push (_stream_readable.js:210:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:166:17) {
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {}
}]
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:30:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:26:17.190] [ERROR] [Default] - Error: MongoNetworkError: failed to connect to server [127.0.0.1:27017] on first connect [MongoError: Authentication failed.
    at Function._getError (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:141:14)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:191:31
    at _callback (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:320:5)
    at Connection.messageHandler (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:349:5)
    at Connection.emit (events.js:200:13)
    at processMessage (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:200:13)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at Socket.Readable.push (_stream_readable.js:210:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:166:17) {
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {}
}]
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:30:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:26:58.440] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:27:39.701] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:29:47.032] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:30:18.096] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:30:58.259] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:31:28.428] [ERROR] [Default] - Error: MongoNetworkError: failed to connect to server [127.0.0.1:27017] on first connect [MongoError: Authentication failed.
    at Function._getError (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:141:14)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:191:31
    at _callback (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:320:5)
    at Connection.messageHandler (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:349:5)
    at Connection.emit (events.js:200:13)
    at processMessage (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:200:13)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at Socket.Readable.push (_stream_readable.js:210:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:166:17) {
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {}
}]
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:30:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:31:39.244] [ERROR] [Default] - Error: MongoNetworkError: failed to connect to server [127.0.0.1:27017] on first connect [MongoError: Authentication failed.
    at Function._getError (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:141:14)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:191:31
    at _callback (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:320:5)
    at Connection.messageHandler (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:349:5)
    at Connection.emit (events.js:200:13)
    at processMessage (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:200:13)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at Socket.Readable.push (_stream_readable.js:210:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:166:17) {
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {}
}]
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:30:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:32:12.479] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:33:01.295] [ERROR] [Default] - Error: MongoNetworkError: failed to connect to server [127.0.0.1:27017] on first connect [MongoError: Authentication failed.
    at Function._getError (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:141:14)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:191:31
    at _callback (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:320:5)
    at Connection.messageHandler (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:349:5)
    at Connection.emit (events.js:200:13)
    at processMessage (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:200:13)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at Socket.Readable.push (_stream_readable.js:210:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:166:17) {
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {}
}]
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:30:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:34:39.586] [ERROR] [Default] - Error: MongoNetworkError: failed to connect to server [127.0.0.1:27017] on first connect [MongoError: Authentication failed.
    at Function._getError (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:141:14)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:191:31
    at _callback (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:320:5)
    at Connection.messageHandler (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:349:5)
    at Connection.emit (events.js:200:13)
    at processMessage (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:200:13)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at Socket.Readable.push (_stream_readable.js:210:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:166:17) {
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {}
}]
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:30:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:35:11.394] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:36:17.950] [ERROR] [Default] - Error: MongoNetworkError: failed to connect to server [127.0.0.1:27017] on first connect [MongoError: Authentication failed.
    at Function._getError (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:141:14)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:191:31
    at _callback (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:320:5)
    at Connection.messageHandler (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:349:5)
    at Connection.emit (events.js:200:13)
    at processMessage (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:200:13)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at Socket.Readable.push (_stream_readable.js:210:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:166:17) {
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {}
}]
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:30:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:36:53.101] [ERROR] [Default] - Error: MongoNetworkError: failed to connect to server [127.0.0.1:27017] on first connect [MongoError: Authentication failed.
    at Function._getError (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:141:14)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/auth/scram.js:191:31
    at _callback (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:320:5)
    at Connection.messageHandler (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connect.js:349:5)
    at Connection.emit (events.js:200:13)
    at processMessage (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:384:10)
    at Socket.<anonymous> (/Users/edz/Desktop/resource/zhblog/node_modules/mongodb/lib/core/connection/connection.js:553:15)
    at Socket.emit (events.js:200:13)
    at addChunk (_stream_readable.js:294:12)
    at readableAddChunk (_stream_readable.js:275:11)
    at Socket.Readable.push (_stream_readable.js:210:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:166:17) {
  name: 'MongoError',
  [Symbol(mongoErrorContextSymbol)]: {}
}]
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:30:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:38:27.343] [ERROR] [Default] - Error: MongooseTimeoutError: Server selection timed out after 30000 ms
    at NativeConnection.<anonymous> (/Users/edz/Desktop/resource/zhblog/models/index.js:33:18)
    at NativeConnection.emit (events.js:200:13)
    at /Users/edz/Desktop/resource/zhblog/node_modules/mongoose/lib/connection.js:772:37
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
[2020-02-02T02:50:11.016] [INFO] [Default] - mongo is opened!
[2020-02-02T02:50:13.292] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"userList":[]}} - ::1 - 15ms
[2020-02-02T02:50:24.548] [INFO] [Default] - mongo is opened!
[2020-02-02T02:50:59.173] [INFO] [Default] - mongo is opened!
[2020-02-02T02:51:05.185] [INFO] [Default] - mongo is opened!
[2020-02-02T02:51:07.320] [INFO] [Default] - mongo is opened!
[2020-02-02T02:51:12.033] [INFO] [Default] - mongo is opened!
[2020-02-02T02:51:15.644] [INFO] [Default] - mongo is opened!
[2020-02-02T02:51:16.525] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"userList":[]}} - ::1 - 15ms
[2020-02-02T02:51:23.457] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"userList":[]}} - ::1 - 2ms
[2020-02-02T02:51:44.486] [INFO] [Default] - mongo is opened!
[2020-02-02T02:51:46.230] [INFO] [Default] - mongo is opened!
[2020-02-02T02:51:48.650] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"userList":[]}} - ::1 - 17ms
[2020-02-02T02:52:09.885] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"userList":[]}} - ::1 - 4ms
[2020-02-02T02:52:30.223] [INFO] [Default] - mongo is opened!
[2020-02-02T02:52:36.343] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"userList":[]}} - ::1 - 14ms
[2020-02-02T02:53:15.037] [INFO] [Default] - mongo is opened!
[2020-02-02T02:53:16.422] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"userList":0}} - ::1 - 16ms
[2020-02-02T02:53:25.018] [INFO] [Default] - mongo is opened!
[2020-02-02T02:53:53.656] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"userList":[]}} - ::1 - 14ms
[2020-02-02T02:53:54.965] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"userList":[]}} - ::1 - 2ms
[2020-02-02T02:53:55.297] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"userList":[]}} - ::1 - 3ms
[2020-02-02T02:54:14.930] [INFO] [Default] - mongo is opened!
[2020-02-02T02:54:32.150] [INFO] [Default] - mongo is opened!
[2020-02-02T02:54:32.217] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"userList":[]}} - ::1 - 18ms
[2020-02-02T02:54:33.461] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"userList":[]}} - ::1 - 2ms
[2020-02-02T02:54:33.966] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"userList":[]}} - ::1 - 2ms
[2020-02-02T02:54:34.249] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"userList":[]}} - ::1 - 3ms
[2020-02-02T02:54:34.570] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"userList":[]}} - ::1 - 4ms
[2020-02-02T02:54:35.452] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"userList":[]}} - ::1 - 2ms
[2020-02-02T02:54:36.294] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"userList":[]}} - ::1 - 2ms
[2020-02-02T02:54:59.718] [INFO] [Default] - mongo is opened!
[2020-02-02T02:56:57.648] [INFO] [Default] - mongo is opened!
[2020-02-02T02:57:42.437] [INFO] [Default] - mongo is opened!
[2020-02-02T03:03:06.166] [INFO] [Default] - mongo is opened!
[2020-02-02T03:03:08.534] [INFO] [Default] - mongo is opened!
[2020-02-02T03:03:21.205] [INFO] [Default] - mongo is opened!
[2020-02-02T03:03:35.883] [INFO] [Default] - mongo is opened!
[2020-02-02T03:03:50.391] [INFO] [Default] - mongo is opened!
[2020-02-02T03:03:51.060] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 14ms
[2020-02-02T03:03:52.351] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 2ms
[2020-02-02T03:03:53.223] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 2ms
[2020-02-02T03:03:57.421] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 1ms
[2020-02-02T03:04:00.671] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 4ms
[2020-02-02T03:04:04.749] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 1ms
[2020-02-02T03:04:06.757] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 1ms
[2020-02-02T03:04:07.135] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 2ms
[2020-02-02T03:04:07.353] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 2ms
[2020-02-02T03:04:07.491] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 2ms
[2020-02-02T03:04:08.042] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 1ms
[2020-02-02T03:04:09.535] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 1ms
[2020-02-02T03:04:10.803] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 1ms
[2020-02-02T03:04:11.488] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 1ms
[2020-02-02T03:04:12.252] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 1ms
[2020-02-02T03:04:15.399] [INFO] [Default] - GET 404 /api/user 请求参数： {} 响应参数：undefined - ::1 - 0ms
[2020-02-02T03:04:18.465] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 1ms
[2020-02-02T03:05:17.375] [INFO] [Default] - mongo is opened!
[2020-02-02T03:05:19.243] [INFO] [Default] - mongo is opened!
[2020-02-02T03:05:19.752] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 14ms
[2020-02-02T03:05:20.856] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 2ms
[2020-02-02T03:06:23.240] [INFO] [Default] - mongo is opened!
[2020-02-02T03:06:26.636] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 12ms
[2020-02-02T03:06:43.464] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 2ms
[2020-02-02T03:07:16.257] [INFO] [Default] - mongo is opened!
[2020-02-02T03:07:21.166] [INFO] [Default] - mongo is opened!
[2020-02-02T03:08:18.859] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 16ms
[2020-02-02T03:12:54.643] [INFO] [Default] - mongo is opened!
[2020-02-02T03:12:55.081] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[{"id":0,"name":"web","description":"web 设计","createTime":"2020-02-01T16:50:17.498Z","updateTime":"2020-02-01T16:50:17.498Z"},{"id":0,"name":"web","description":"web 设计","createTime":"2020-02-01T18:53:43.676Z","updateTime":"2020-02-01T18:53:43.676Z"}]}} - ::1 - 38ms
[2020-02-02T03:17:59.020] [INFO] [Default] - mongo is opened!
[2020-02-02T03:18:00.268] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 14ms
[2020-02-02T03:18:10.956] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 2ms
[2020-02-02T03:18:11.980] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 1ms
[2020-02-02T03:18:12.294] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 1ms
[2020-02-02T03:18:13.051] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 1ms
[2020-02-02T03:18:13.368] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 2ms
[2020-02-02T03:18:38.635] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 1ms
[2020-02-02T03:19:55.168] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 2ms
[2020-02-02T03:19:56.062] [INFO] [Default] - mongo is opened!
[2020-02-02T03:19:56.394] [ERROR] [Default] - TypeError: Cannot read property 'find' of undefined
    at Object.getCategoryList (/Users/edz/Desktop/resource/zhblog/services/category.js:15:36)
    at getCategoryList (/Users/edz/Desktop/resource/zhblog/controllers/category.js:15:52)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:353:16
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:358:31)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-router/lib/router.js:339:32)
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at /Users/edz/Desktop/resource/zhblog/node_modules/koa-static-cache/index.js:62:42
    at dispatch (/Users/edz/Desktop/resource/zhblog/node_modules/koa-compose/index.js:42:32)
    at bodyParser (/Users/edz/Desktop/resource/zhblog/node_modules/koa-bodyparser/index.js:86:11)
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
    at async loggerMiddleware (/Users/edz/Desktop/resource/zhblog/middlewares/logger.js:48:5)
[2020-02-02T03:19:56.396] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":-1,"data":null,"msg":"Cannot read property 'find' of undefined"} - ::1 - 3ms
[2020-02-02T03:20:03.662] [INFO] [Default] - mongo is opened!
[2020-02-02T03:20:16.614] [INFO] [Default] - mongo is opened!
[2020-02-02T03:20:18.353] [INFO] [Default] - mongo is opened!
[2020-02-02T03:20:19.401] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 14ms
[2020-02-02T03:20:20.463] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[]}} - ::1 - 2ms
[2020-02-02T03:20:33.299] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[{"id":0,"name":"web","description":"web 设计","createTime":"2020-02-01T16:50:17.498Z","updateTime":"2020-02-01T16:50:17.498Z"},{"id":0,"name":"web","description":"web 设计","createTime":"2020-02-01T18:53:43.676Z","updateTime":"2020-02-01T18:53:43.676Z"}]}} - ::1 - 22ms
[2020-02-02T03:20:35.051] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[{"id":0,"name":"web","description":"web 设计","createTime":"2020-02-01T16:50:17.498Z","updateTime":"2020-02-01T16:50:17.498Z"},{"id":0,"name":"web","description":"web 设计","createTime":"2020-02-01T18:53:43.676Z","updateTime":"2020-02-01T18:53:43.676Z"}]}} - ::1 - 3ms
[2020-02-02T03:21:26.498] [INFO] [Default] - mongo is opened!
[2020-02-02T03:21:40.383] [INFO] [Default] - mongo is opened!
[2020-02-02T03:21:49.805] [INFO] [Default] - mongo is opened!
[2020-02-02T03:22:13.930] [INFO] [Default] - mongo is opened!
[2020-02-02T03:22:20.098] [INFO] [Default] - mongo is opened!
[2020-02-02T03:22:22.726] [INFO] [Default] - mongo is opened!
[2020-02-02T03:22:29.904] [INFO] [Default] - mongo is opened!
[2020-02-02T03:22:37.570] [INFO] [Default] - mongo is opened!
[2020-02-02T03:22:39.769] [INFO] [Default] - mongo is opened!
[2020-02-02T03:22:44.205] [INFO] [Default] - mongo is opened!
[2020-02-02T03:22:50.391] [INFO] [Default] - mongo is opened!
[2020-02-02T03:22:53.589] [INFO] [Default] - mongo is opened!
[2020-02-02T03:23:03.386] [INFO] [Default] - mongo is opened!
[2020-02-02T03:23:20.143] [INFO] [Default] - mongo is opened!
[2020-02-02T03:23:43.426] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[{"id":0,"name":"web","description":"web 设计","createTime":"2020-02-01T16:50:17.498Z","updateTime":"2020-02-01T16:50:17.498Z"},{"id":0,"name":"web","description":"web 设计","createTime":"2020-02-01T18:53:43.676Z","updateTime":"2020-02-01T18:53:43.676Z"}]}} - ::1 - 32ms
[2020-02-02T03:23:45.601] [INFO] [Default] - GET 200 /api/users 请求参数： {} 响应参数：{"code":200,"msg":"","data":{"categoryList":[{"id":0,"name":"web","description":"web 设计","createTime":"2020-02-01T16:50:17.498Z","updateTime":"2020-02-01T16:50:17.498Z"},{"id":0,"name":"web","description":"web 设计","createTime":"2020-02-01T18:53:43.676Z","updateTime":"2020-02-01T18:53:43.676Z"}]}} - ::1 - 3ms
[2020-02-02T03:24:45.712] [INFO] [Default] - mongo is opened!
